db/migrate/20180911160324_devise_create_users.rb

```diff
@@ -6,8 +6,8 @@ def change
       ## Database authenticatable
       t.string :email,              null: false, default: ""
       t.string :encrypted_password, null: false, default: ""
-      t.string :first_name, null: false, default: ""
-      t.string :last_name, null: false, default: ""
+      t.string :first_name, null: false
+      t.string :last_name, null: false
```

Comment:
Oh, i forgot: if you add this here (null: false) you should also add a validation to the model the checks for this.

Check this: https://guides.rubyonrails.org/active_record_validations.html The validation that checks for `presence true`